pages/home/changelog.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Changelog"
+description: "What's new at Arcade.dev"
+---
+
+# Changelog
+
+*Here's what's new at Arcade.dev!*
+
+## What's new for the week ending on 2025-06-20
+
+**Frameworks**
+* `[feature - 🚀]` Support and examples for OpenAI Agent SDK in Typescript ([link](https://docs.arcade.dev/home/oai-agents/overview))
+
+**Toolkits**
+* `[feature - 🚀]` Jira toolkit released ([link](https://docs.arcade.dev/toolkits/productivity/jira))
+
+**CLI and TDK**
+* `[feature - 🚀]` V2.0 of TDK ([link](https://docs.arcade.dev/home/migrate-to-v2))
+* `[feature - 🚀]` Admin API client support (Update to v1.6.0 of `arcade-py`, or v1.8.0 of the `arcade-js`, or v0.1.0-alpha.4 of `arcade-go` package)
+
+**Platform and Engine**
+* `[feature - 🚀]` Admin APIs released for managing users, secrets, and tools ([API References](https://reference.arcade.dev/api-reference#tag/admin))
+* `[bugfix - 🐛]` Unauthenticated MCP servers can be called anonymously
+* `[feature - 🚀]` End-user credentials and auth status can be fetched in batches ([docs](https://docs.arcade.dev/home/auth/tool-auth-status))
```

Comment:
```suggestion
* **🚀 New feature:** Admin APIs released for managing users, secrets, and tools ([API References](https://reference.arcade.dev/api-reference#tag/admin))
* **🐛 Bug fix:** Unauthenticated MCP servers can be called anonymously
* **🚀 New feature:** End-user credentials and auth status can be fetched in batches ([docs](https://docs.arcade.dev/home/auth/tool-auth-status))
```

This would be a bit more readable for me, but use your judgement/preference!